x-pack/plugins/cloud_security_posture/public/pages/findings/layout/findings_group_by_selector.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useMemo } from 'react';
+import { EuiComboBox, EuiFormLabel, EuiSpacer, type EuiComboBoxOptionOption } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useHistory } from 'react-router-dom';
+import { i18n } from '@kbn/i18n';
+import { INTERNAL_FEATURE_FLAGS } from '../../../../common/constants';
+import type { FindingsGroupByKind } from '../types';
+import { findingsNavigation } from '../../../common/navigation/constants';
+
+const getGroupByOptions = (): Array<EuiComboBoxOptionOption<FindingsGroupByKind>> => [
```

Comment:
nit: i don't mind functions to get static data but I think this one can definitely just be an array

Reply:
i kept it as a function so it'll be evaluated on the component level (mount in this case) and not on the module level, assuming it means something to `locale` changes (but if no, then yeah for sure)